src/js/CheckboxTree.js

```diff
@@ -190,13 +190,14 @@ class CheckboxTree extends React.Component {
             const key = `${node.value}`;
             const checked = this.getCheckState(node, noCascade);
             const children = this.renderChildNodes(node);
+            const nodeDisabled = !!(disabled || node.disabled);
```

Comment:
Yes. The disabled property on TreeNode isRequired. The double negation will transform undefined to false.